test/integration/user_membership_test.rb

```diff
@@ -0,0 +1,67 @@
+require "test_helper"
```

Comment:
Will it be possible to split the method in different tests?

Reply:
different functions so if the first assert fails, we still test the others ones.
So we know if a regression affects all features, or just one.